nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -1,14 +1,55 @@
 # Release 22.11 (“Raccoon”, 2022.11/??) {#sec-release-22.11}
 
-Support is planned until the end of June 2023, handing over to 23.05.
+This release is supported until the end of June 2023, handing over to NixOS 23.05.
 
 ## Highlights {#sec-release-22.11-highlights}
 
-In addition to numerous new and upgraded packages, this release has the following highlights:
+In addition to numerous new and upgraded packages, this release includes the following highlights:
 
 - GNOME has been upgraded to 43. Please take a look at their [Release
   Notes](https://release.gnome.org/43/) for details.
 
+- KDE Plasma has been upgraded to 5.25 and 5.26.
+    - https://kde.org/announcements/plasma/5/5.25.0/
+    - https://kde.org/announcements/plasma/5/5.26.0/
```

Comment:
We should probably only mention the latest version here. cc @K900

Reply:
The changelogs aren't cumulative, so you'll have to read both if you want to figure out what changed.

Reply:
Yes, we can say "upgraded from 5.24 to 5.26" and then link both changelogs.